src/main/resources/application-standalone.conf

```diff
@@ -37,6 +37,7 @@ agreement-process {
     user-registry                  = "https://gateway.interop.pdnd.dev/user-registry/0.1"
   }
 
+  events-queue-name         = ${EVENTS_QUEUE_NAME}
```

Comment:
[same](https://github.com/pagopa/interop-be-agreement-process/pull/238/files#r1253046683)